dvc/stage.py

```diff
@@ -299,7 +307,11 @@ def remove_outs(self, ignore_remove=False, force=False):
                         out=out, stage=self.relpath
                     )
                 )
-                out.remove(ignore_remove=ignore_remove)
+
+                if force:
+                    ask = False
+
+                out.remove(ignore_remove=ignore_remove, force=not ask)
```

Comment:
all this naming is super confusing - force, aks, confirm mean different things even in the same function sometimes

Reply:
yep, can't agree more, @shcheklein , but changing this will require a big refactor, `force` means different things, for example: `ask` / `prompt` / `confirm` are different names for the same thing, the problem is that `prompt` is the module, `confirm` is the method and `_ask` is the implementation beneath `confirm` :sweat_smile:

We can address this in another issue.

Reply:
@mroutis Seems like `force` has the meaning of something like `remove_persistent`? Maybe let's switch them up? Your `ask` -> `force` and `force` -> `remove_persistent`. Would that make sense?

Reply:
@efiop , lol, it make sense, I'll do that ^